flask_rebar/swagger_generation/swagger_generator_v3.py

```diff
@@ -236,7 +238,12 @@ def _get_paths(self, paths, default_headers_schema, default_security=None):
                 }
 
                 if d.func.__doc__:
-                    path_definition[method_lower][sw.description] = d.func.__doc__
+                    docstring = chomp_right(chomp_left(d.func.__doc__))
```

Comment:
Do you mind explaining why `strip` is not sufficient here? It's not clear to me.